x-pack/filebeat/input/s3/_meta/fields.yml

```diff
@@ -0,0 +1,19 @@
+- key: s3
+  title: "s3"
+  description: >
+    Fields from s3 input.
+  release: beta
+  fields:
+    - name: log.source
+      type: group
+      description: >
+        Source of the log message.
+      fields:
+        - name: bucket_name
```

Comment:
I see you use `s3_request.bucket.name` and `s3_daily_storage.bucket.name` in the Metricbeat module. Having all these 3 under the same field would make a lot of sense. WDYT about putting them under `aws.s3.bucket.name`? That gives the user the opportunity to correlate logs and metrics based on this field.

I would not change all of them at the same time, but decide on one now and use it here, think about changing the metricsets later

Reply:
@exekias Yeah good point! I did not think this far 😂 I think in s3 metricset should both be `s3.bucket_name`. WDYT? I don't know why I called it `bucket.name` instead of `bucket_name` 😬